fedmsg_meta_fedora_infrastructure/greenwave.py

```diff
@@ -0,0 +1,83 @@
+# This file is part of fedmsg.
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# fedmsg is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# fedmsg is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with fedmsg; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#
+# Authors:  Ralph Bean <rbean@redhat.com>
+#
+
+try:
+    # Python3
+    from urllib.parse import urlencode
+except ImportError:
+    # Python2
+    from urllib import urlencode
+
+from fedmsg_meta_fedora_infrastructure import BaseProcessor
+
+
+class GreenwaveProcessor(BaseProcessor):
+    __name__ = "greenwave"
+    __description__ = "Greenwave Gating Service"
+    __link__ = "greenwave-web-greenwave.app.os.fedoraproject.org/api/v1.0/policies"
+    __docs__ = "https://fedoraproject.org/wiki/Infrastructure/Factory2/Focus/Greenwave"
+    __obj__ = "Gating Decisions"
+    # XXX - We have a greenwave icon in the works.  Let's use this for now.
```

Comment:
In my opinion it's better to write an issue for this sort of thing rather than a TODO